### ISSUE TITLE
A dead end with the Chocolatey sketchup package?

### ISSUE THREAD

**Issue by fds39** [pal1000/mesa-dist-win#45]
I'm facing the unsupported Hardware acceleration error message that your great software responds to.
I installed Sketchup 2017 as a Chocolatey package.

2 remarks : 
- This 2017 release of SketchUp is the last, installable, free version of SketchUp.
- the Chocolatey page mentions the Hardware requirement : it requires a 3D class video card with 1GB of memory or higher and hardware acceleration which my old Dell Latitude laptop does not have.

Here is [the dedicated page ](https://chocolatey.org/packages/sketchup)
Once all the information required by mesa3d was filled in, I launched sketup.exe
But nothing happens anymore. So I sadly had to uninstall the package.

I don't know if the blocking is on the side of this old 2017 Sketchup version or if we I should rather study the specificity of the Chocolatey package.

Any idea ?

**Comment by pal1000** [comment]
Installed Sketchup to reproduce this but nothing unusual happened. Note that it's a 64-bit program. I deployed Mesa desktop OpenGL 64-bit in `C:\Program Files\SketchUp\SketchUp 2017` with `perappdeployment.cmd` and that was enough to convince Sketchup to use Mesa.
![Screenshot 2021-01-24 010108](https://user-images.githubusercontent.com/1138235/105616523-685e4c80-5de0-11eb-8320-14ffb069d561.jpg)

**Comment by fds39** [comment]
Well I don't really have much to add, because I proceeded identically with perappdeployment.cmd 

In fact I tested on a second similar laptop Dell latitude 4300 (really old) with the same result: 
By running C:\Program Files\SketchUp\SketchUp 2017\SketchUp.exe, the computer remains inert: not the slightest trace of sketchup in the task manager.

My laptop dates from 2008: maybe the explanation ...

Thanks for your help
The steps I followed https://i.imgur.com/ExLg3oN.png

**Comment by pal1000** [comment]
I looked at your screenshot. I see you filled in the executable filename in per app deployment. I don't advise doing that unless you tried without first and program didn't work because currently it sticks even if you don't supply itt again. To reverse it's effect please delete `C:\Program Files\SketchUp\SketchUp 2017\SketchUp.exe.local`.

Also graw and swr aren't required to make it work.

I'll update per app deployment tool to wipe `*.exe.local` files from specified folder if no filename is specified. This should help in situations like yours.

**Comment by fds39** [comment]
Yes I tried without first and program didn't work. Deleting `C:\Program Files\SketchUp\SketchUp 2017\SketchUp.exe.local` didn't change anything either.
I started from scratch by following your instructions to the letter but the result is not better.
https://i.imgur.com/HpIaiSr.png

**Comment by pal1000** [comment]
Did you use 20.3.3 MinGW package? If so try 20.3.3 MSVC release. It has lower CPU requirements.
Maybe your CPU is so old that it can't handle [SSSE3](https://en.wikipedia.org/wiki/SSSE3#CPUs_with_SSSE3).

**Comment by fds39** [comment]
From what I have read, DELL Latitude E4300 Intel Core 2 Duo P9400 processor includes SSSE3 and SSE4.1 instructions.

For your information I learned about your software creation through [this video tutorial](https://youtu.be/surKa7QFDBw).
The 20.3.3 MSVC release is the only version I had tested until yesterday. By conscience, I tested 20.3.3 MSVC release, unfortunately without any more result.
I think you tried everything to help me out and I thank you very much for that. 

We will try with my 10 years old daughter to use the free web version of SketchUp for her "solar oven mini-project" in technology class ;)
